packages/app-page-builder/src/types.ts

```diff
@@ -22,6 +22,20 @@ export enum PageImportExportTaskStatus {
     COMPLETED = "completed",
     FAILED = "failed"
 }
+
+// TODO: for Webiny core team: create this type in the app-file-manager
+export interface File {
+    id: string;
+    name: string;
+    key: string;
+    src: string;
+    size: number;
+    type: string;
+    tags: string[];
+    meta: Record<string, any>;
+    createdOn: string;
+}
```

Comment:
@neatbyte-vnobis I updated this type based on the actual GraphQL query being executed, as API types do not match entirely (not all fields are returned from the API). This is the actual data that you get from the API.